### PR TITLE
Link

### PR DESCRIPTION
**Pull request by HarshitSachdeva**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -397,6 +397,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :books: [Docker Curriculum Ebook](https://docker-curriculum.com/)
 - :wrench: [Docker Labs environment](https://labs.play-with-docker.com/)
 - :books: [Docker labs](https://www.sharelearn.net/practice/dockerlabs)
+- :books: [Docker Cheat sheet](https://dockerlabs.collabnix.com/docker/cheatsheet/)
```
Comment (shahednasser) [inline README.md:400]:
```suggestion
- :bulb: [Docker Cheat sheet](https://dockerlabs.collabnix.com/docker/cheatsheet/)
```

**Comment by HarshitSachdeva** [comment]
@shahednasser Done. Pls merge.

**Comment by HarshitSachdeva** [comment]
@shahednasser I have made changes according to suggestions. Pls merge.

**Review by shahednasser — APPROVED** [review]

**Comment by shahednasser** [comment]
Thanks!

**Pull request merged**